pytentiostat/LSV-v0.1.py

```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf-8 -*-
+
+from pyfirmata import Arduino, util
+from matplotlib import pyplot as plt
+import time
+import numpy as np
+import pandas as pd
+import serial.tools.list_ports
+
+print('Welcome to the JUAMI pytentiostat interface!')
+print('Searching for potentiostat...')
+
+ports = list(serial.tools.list_ports.comports())  # get list of serial ports connected
+arduinos = 0
+for p in ports:  # Checking for Arduino Unos connected
+  if 'Arduino Uno' in p.description:
+    com = p.device
+    arduinos += 1
+
+if arduinos > 1:  # If two Arduino Unos are found, program exits
+  print('More than one Arduino Uno found. Exiting...')
+  time.sleep(5)
+  exit()
+
+if arduinos == 0:  # If no Arduino Uno is found, program exits
+  print('No JUAMI potentiostat found. Exiting...')
+  time.sleep(5)
+  exit()
+
+
+try:
+  board = Arduino(com)  # opens communication to Arduino
+except:
+  print('Error. Could not open COM port')
+  time.sleep(5)
+  exit()
+print('Potentiostat connected (' + com + '). Reading configuration file...')
+try:
+  time.sleep(0)  # Place holder until config file is finished
+  # config = yaml.load(file)
+except:
+  print('Could not read file. Exiting...')
+  time.sleep(5)
+  exit()
+
+
+print('Configuration file loaded successfully. Beginning experiment...')
+'----------------------------------------------------------------------------'
+'----------------------------------------------------------------------------'
+# User defined variables
+quiet_time = 5.0  # specify time for arduino to wait before recording data.
+                  # This removes charging current if desired by the user
+start_V = 0.74
```

Comment:
all non-global variables should be lower-case, so this should probably be ``start_voltage``.  It is more typing, but it will also make the code more readable.  Using case correctly according to PEP8 standards is very helpful for interpretability is also helpful.  Classes have first letter capitalized, modules and functions all lower case, global variables all upper case.